src/components/filterableproducttable/FilterableProductTable.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import SearchBar from '../searchbar/SearchBar'
+import ProductTable from '../producttable/ProductTable'
+import productData from '../../data.json'
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+export default class FilterableProductTable extends Component {
```

Comment:
Your lab looks awesome mate! Just missing the red part of the products out of stock. Nevertheless, I think it was a personal option. Very good job! 🥇